src/frontend/admin/src/layouts/dashboard/nav/header/DashboardNavHeader.tsx

```diff
@@ -45,8 +43,8 @@ export function DashboardNavHeader() {
           href={PATH_ADMIN.rootAdmin}
           title={intl.formatMessage(message.goBackToHome)}
         >
-          <Image
-            src={logo}
+          <img
+            src="images/logo/logo-fun.svg"
```

Comment:
I'll use absolute path here. `/images/logo/logo-fun.svg`.

Furthermore, I'll continue to use `Image` next component. Then, I'll just disable image optimization from `next.config.js`. In this way, later, if we want to turn on image optimization we just have to tweak next config not refactor our components.

```ts
const nextConfig = {
  output: "export",
  images: {
    // Disable image optimization, as the default image optimizer does not support static export
    unoptimized: true,
  },
  ...
```